test/passing/refs.ocamlformat/max_indent.ml.ref

```diff
@@ -7,7 +7,7 @@ let () =
 let () =
   fooooo
   |> List.iter
-       (fun some_really_really_really_long_name_that_doesn't_fit_on_the_line ->
+    (fun some_really_really_really_long_name_that_doesn't_fit_on_the_line ->
     let x =
```

Comment:
Not exactly great, but the exact same thing already happens when doing `map ~f (fun` without `|>`.

Reply:
I added a test to showcase this.